deployer/hub.py

```diff
@@ -71,22 +68,28 @@ def auth_gcp(self):
         location = config.get('zone', config.get('region'))
         cluster = config['cluster']
         with tempfile.NamedTemporaryFile() as kubeconfig:
-            with decrypt_file(key_path) as decrypted_key_path:
+            orig_kubeconfig = os.environ.get('KUBECONFIG')
```

Comment:
Oh wow... You caught this because KUBECONFIG wasn't used, right?
Hooray to having a linter now :tada:

Reply:
EXACTLY! It was messing with my local credentials too